File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/recyclerView/selector/MeteoriteSelectorPortrait.java
package com.antonio.samir.meteoritelandingsspots.ui.recyclerView.selector;


public class MeteoriteSelectorPortrait implements MeteoriteSelector {

    private final MeteoriteSelectorView view;

    public MeteoriteSelectorPortrait(MeteoriteSelectorView view) {
        this.view = view;
    }

    @Override
    public void selectItemId(final String meteorite) {
        view.selectPortrait(meteorite);
    }
}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/recyclerView/selector/MeteoriteSelectorPortrait.kt
package com.antonio.samir.meteoritelandingsspots.ui.recyclerView.selector


class MeteoriteSelectorPortrait(private val view: MeteoriteSelectorView) : MeteoriteSelector {

    override fun selectItemId(meteorite: String?) {
        view.selectPortrait(meteorite)
    }
}
